index.js

```diff
@@ -90,17 +93,34 @@ module.exports = {
     options.removeFilters = [ 'published' ].concat(options.removeFilters || []);
 
   },
-  
+
   afterConstruct: function(self, callback) {
+    self.enableCloudStatic();
     self.composeDeployTo();
     self.excludeFromWorkflow();
     self.addRoutes();
     self.apos.pages.addAfterContextMenu(self.menu);
+    self.apos.pages.addAfterContextMenu(self.visualDiff);
     self.addCsrfExceptions();
+
+    self.pushAsset('script', 'visual-diff', {
+      when: 'user'
+    });
     return self.ensureIndexes(callback);
   },
 
   construct: function(self, options) {
+    var backstopConfig = path.join(__dirname, 'backstop-config.json');
+    backstopConfig = JSON.parse(fs.readFileSync(backstopConfig));
+
+    if (options.backstopConfig && options.backstopConfig.viewports) {
+      backstopConfig.viewports = _.concat(backstopConfig.viewports, options.backstopConfig.viewports)
+    }
+
+    if (options.backstopConfig && options.backstopConfig.scenarios) {
+      _.merge(backstopConfig.scenarios[0], options.backstopConfig.scenarios);
```

Comment:
Why would you merge what sounds like an array into a single element in another array? Does that make sense?